### PR TITLE
ui: list unvaulting vaults and add revault action

### PR DESCRIPTION
**Pull request by edouardparis**
based on #58 
waiting cancel feature on revaultd

### CODE REVIEW

File: src/ui/view/vault.rs [hunk 1]
```diff
@@ -223,9 +223,57 @@ impl VaultOnChainTransactionsPanel {
                             .align_items(Align::Center),
                     )))
                 }
+                VaultStatus::Unvaulted | VaultStatus::Unvaulting => {
+                    col = col.push(card::white(Container::new(
+                        Row::new()
+                            .push(
+                                Container::new(text::simple(
+                                    "Funds are moving, do you want to revault it ?",
```
Comment (danielabrozzoni) [inline src/ui/view/vault.rs:231]: nit: maybe "do you want to revault *them?*" (also please remove the space before the ?)

File: src/ui/state/vault.rs [hunk 2]
```diff
@@ -150,6 +153,15 @@ pub enum VaultSection {
         view: AcknowledgeVaultView,
         signer: SignState,
     },
+    /// Revault action ask the user if the vault that is unvaulting
+    /// should be revolted and executes the revault command after
```
Comment (danielabrozzoni) [inline src/ui/state/vault.rs:157]: nit: "revaulted"?

**Comment by edouardparis** [comment]
tested by the :smiling_imp: gang they didnt ACK but I am confident
ACK 8f03195d7d218339c0cef7ca108bf3528d694d45

**Comment by darosior** [comment]
Didn't review but ACK merging it if you are confident, we'll test tonight anyways.

**Pull request merged**